### ISSUE TITLE
Icon request: Unsplash

### ISSUE THREAD

**Issue by jamesponddotco** [feathericons/feather#546]
## Icon Request

* **Icon name:** Unsplash
* **Use case:** Same as most social networks, to link to your own profile on the website
* **Screenshots of similar icons:**

<img width="654" alt="screen shot 2019-02-21 at 13 15 22" src="https://user-images.githubusercontent.com/5659117/53183813-c7a2f900-35da-11e9-8c41-b1e399dc3a6c.png">

**Comment by ashygee** [comment]
Unsplash Icon
<img width="29" alt="screenshot 2019-03-02 20 00 00" src="https://user-images.githubusercontent.com/10384315/53690775-cdf54b80-3d25-11e9-9a02-9b32a90ce88a.png">
Side-by-side example
<img width="98" alt="screenshot 2019-03-02 19 59 48" src="https://user-images.githubusercontent.com/10384315/53690778-d188d280-3d25-11e9-8a8f-85cb1b38fcc6.png">

@colebemis what do you think?

**Comment by colebemis** [comment]
Looking good! This is a deceptively hard one. I'd like to avoid aligning straight lines to half-pixels if we can: 
![image](https://user-images.githubusercontent.com/4608155/53704938-bf666d00-3dd5-11e9-9fa9-51c99ee543b0.png)

What do you think of this version?
![image](https://user-images.githubusercontent.com/4608155/53704981-0eac9d80-3dd6-11e9-8a88-1c3d70302272.png)

**Comment by ashygee** [comment]
@colebemis yeah that works. I made something similar but wasn't sure about the square/rectangular-ness of the detached piece.

**Comment by colebemis** [comment]
@ashygee Oh good point. What do you think of these two versions? And which do you prefer?

![image](https://user-images.githubusercontent.com/4608155/54774159-513ceb00-4bc8-11e9-854d-1fe4df4fe922.png)

**Comment by MarcelloTheArcane** [comment]
See #763
